enatega-multivendor-app/src/screens/Menu/Menu.js

```diff
@@ -270,22 +261,42 @@ function Menu({ route, props }) {
     else {
       return (
         <View style={styles().emptyViewContainer}>
-          <TextDefault textColor={currentTheme.fontMainColor}>
-            {t('noRestaurants')}
-          </TextDefault>
+          <View style={styles().emptyViewBox}>
+            <TextDefault bold H4 center textColor={currentTheme.fontMainColor}>
+              We are not currently available in your area
```

Comment:
If translation is necessary for current release then add text in translation file and use, if not required yet, then leave as it is.